### ISSUE TITLE
10_vwm-multi.pdf

### ISSUE THREAD

**Issue by PeterKolarovec** [mskl/fit-bi-szz#4]
Treshold algoritmus:
- nekončí sa pri nájdení K objektov, ale ak výsledok po agregačnej funkcií K+1 objektu je horší ako prvku K, inak K+1 objekt nahradí objekt K, opakuje sa pokiaľ nenarazíme na treshold

**Comment by mskl** [comment]
Treshold skončí když hodnota agregační fce na i-tým řádku nad seřazenýma parametrama je < než skóre objektů, který jsem v tom řádku našel (a přes random access k nim přistoupil), ne? :)

viz: http://alumni.cs.ucr.edu/~skulhari/Top-k-Query.pdf

**Comment by PeterKolarovec** [comment]
Áno, tak. Ale z handoutu som mal pocit, že @pauknvoj nastavuje treshold explicitne na 0.7 a chce skončiť hneď po nájední k objektov.
